command/sub8_launch/nodes/self_check.py

```diff
@@ -136,6 +146,20 @@ def tx_init(self):
     @txros.util.cancellableInlineCallbacks
     def do_check(self):
         passed = True
+
+        # Check if front cameras are actally on usb bus
+        command = "lsusb -d {}".format(self.front_cam_product_id)
+        err_str = "{} front camera{} not connected to usb port"
+        try:
+            count_front_cam_usb = \
+                subprocess.check_output(["/bin/sh", "-c", command]).count("Point Grey")
```

Comment:
Do you have to have the first two commands here? Won't just calling lsusb -d work?